.circleci/config.yml

```diff
@@ -66,14 +69,14 @@ common-steps:
       command: |
         cd ~/packaging/securedrop-debian-packaging
         export PKG_VERSION=1000.0
-        export PKG_PATH=/home/circleci/project/dist/securedrop-client-$PKG_VERSION.tar.gz
+        export PKG_PATH=~/project/dist/securedrop-client-$PKG_VERSION.tar.gz
         make securedrop-client
 
 version: 2
 jobs:
   build-buster:
     docker:
-      - image: circleci/python:3.7-buster
+      - image: python:3.7-buster
```

Comment:
:bulb: Are these "upstream"/Docker-maintained images?

Reply:
They're upstream of us, yes, hosted here: https://hub.docker.com/_/python and source code is here: https://github.com/docker-library/python I'd be just as happy using `debian:buster`, but then we'd have to `apt-get install python3` in every CI job.

Reply:
I was wondering why the image wouldn't show up in docker hub when you searched for the tag `3.7-buster`. Looks like they're keeping these legacy images hidden from search results.

It is kinda nice how simple the [buster dockerfile](https://github.com/debuerreotype/docker-debian-artifacts/blob/d5eb7c589d016973bce6f3e1827b5c315b7cefbc/buster/Dockerfile) is compared to [the python one](https://github.com/docker-library/python/blob/a7655831c15e823dffd1c895e87d66edfeadebb7/3.7/buster/Dockerfile), so I have a slight preference for going the route of the debian image + `apt-get install python3`. I just think it's a bit clearer what the exact environment requirements are this way.

Reply:
@conorsch I can review if you prefer to keep the pr as is

Reply:
> I was wondering why the image wouldn't show up in docker hub when you searched for the tag 3.7-buster

If you ctrl+f for `3.7-buster` on https://hub.docker.com/_/python, it's there. The web search on docker hub must not support tags! 

> It is kinda nice how simple the buster dockerfile is compared to the python one, so I have a slight preference for going the route of the debian image + apt-get install python3

Sounds nice to me, then we'd be in a better position to know definitely what's required outside the base Debian stable install, which is a good thing on all counts, if you don't mind the extra verbosity. Go ahead and check it out! Tip: if you want to debug locally, try `circleci local execute --job <job>`, then you don't have to push every time. =)